@here/harp-mapview/lib/AnimatedExtrusionHandler.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TileKey } from "@here/harp-geoutils";
+import { DataSource } from "./DataSource";
+import { MapView } from "./MapView";
+import { Tile } from "./Tile";
+
+/**
+ * Animation states for extrusion effect
+ */
+export enum AnimatedExtrusionState {
+    None,
+    Started,
+    Playing,
+    Finished
+}
+
+/**
+ * Creates `AnimatedExtrusionHandler` that controls animated extrusion effect for
+ * extruded objects in [[Tile]] when they appear on the scene.
```

Comment:
not in `Tile` but in `MapView`.

Albeit as written above, this code begs for `AnimatedExtrusionTileHandler`